### ISSUE TITLE
TypeError: int() argument must be a string or a number, not 'datetime.datetime'

### ISSUE THREAD

**Issue by zerc** [benoitbryon/django-downloadview#104]
Hello! 

Some times i catched this exception when downloaded file. 
After some digging i found why.

This happens when request has  ``If-Modified-Since`` header.

Exception raised here: 
https://github.com/benoitbryon/django-downloadview/blob/master/django_downloadview/views/base.py#L119

```python
def was_modified_since(self, file_instance, since):
        """Return True if ``file_instance`` was modified after ``since``.
        Uses file wrapper's ``was_modified_since`` if available, with value of
        ``since`` as positional argument.
        Else, fallbacks to default implementation, which uses
        :py:func:`django.views.static.was_modified_since`.
        Django's ``was_modified_since`` function needs a datetime and a size.
        It is passed ``modified_time`` and ``size`` attributes from file
        wrapper. If file wrapper does not support these attributes
        (``AttributeError`` or ``NotImplementedError`` is raised), then
        the file is considered as modified and ``True`` is returned.
        """
        try:
            return file_instance.was_modified_since(since)
        except (AttributeError, NotImplementedError):
            try:
                modification_time = file_instance.modified_time
                size = file_instance.size
            except (AttributeError, NotImplementedError):
                return True
            else:
                return was_modified_since(since, modification_time, size)
```

because ``modification_time`` it is realy datetime object for default storage (FileSystemStorage)

https://github.com/django/django/blob/master/django/core/files/storage.py#L324

``` python
    def modified_time(self, name):
        return datetime.fromtimestamp(os.path.getmtime(self.path(name)))
```

I can to contribute if needed.

**Comment by benoitbryon** [comment]
Thanks for the report!

> I can to contribute if needed.

It would be very nice :)

**Comment by zerc** [comment]
Oh sorry for this spam

**Comment by benoitbryon** [comment]
Hi @zerc and @benesch ! I merged your pull-requests and released django-downloadview 1.8 :)
Thank you!

**Comment by benesch** [comment]
Thanks so much! Looks great.
